### PR TITLE
Fix GH-2474

### PR DESCRIPTION
**Pull request by phit**
# Description

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

* [x] You verify that the start command applied does not use a shell script
  * [x] If some script is needed then it is part of a current yolk or a PR to add one
* [x] The egg was exported from the panel

### CODE REVIEW

File: bots/discord/jmusicbot/egg-j-music-bot.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
-            "container": "debian:bullseye-slim",
+            "container": "ghcr.io\/pterodactyl\/installers:debian",
```
Comment (QuintenQVD0) [inline bots/discord/jmusicbot/egg-j-music-bot.json:26]: Please leave this at debian 11

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: bots/discord/jmusicbot/egg-j-music-bot.json (hunk 1 above)
Comment (phit) [reply bots/discord/jmusicbot/egg-j-music-bot.json:26]: why?

Comment (phit) [reply bots/discord/jmusicbot/egg-j-music-bot.json:26]: should i change this to `ghcr.io\/parkervcp\/installers:debian` instead?

Comment (phit) [reply bots/discord/jmusicbot/egg-j-music-bot.json]: makes sense, fixed

**Comment by QuintenQVD0** [comment]
It can  stay java 17 if that id needed?

**Comment by phit** [comment]
it's not needed, the jar runs fine even with Java 8, but Java 17 is a LTS release, while 16 has been [EOL](https://endoflife.date/java) for a while, also _most_ people will already have that 17 image for Minecraft

**Comment by QuintenQVD0** [comment]
> it's not needed, the jar runs fine even with Java 8, but Java 17 is a LTS release, while 16 has been [EOL](https://endoflife.date/java) for a while, also _most_ people will already have that 17 image for Minecraft

If it works with 17 then wy did you revert back to 16?

**Comment by phit** [comment]
I didn't?

**Comment by QuintenQVD0** [comment]
> I didn't?

Sorry! Long day. The gir diff is confusing me.

**Comment by phit** [comment]
No worries, thanks for the fast merge!

**Pull request merged**